docs/src/reference/configuration.rst

```diff
@@ -141,410 +168,513 @@ title
-~~~~~~~~
+.. code:: yaml
 
--  type: integer
--  description: The minimum number of tips a clade must have before its logistic growth is calculated.
--  default: ``50``
+     - strain
+     - name
+     - "Virus name"
 
-min_frequency
+database_id_columns
+~~~~~~~~~~~~~~~~~~~
+
+-  type: object
+-  description: A list of columns representing external database ids for metadata records. These unique ids represent a snapshot of data at a specific time for a given strain name. The sanitize metadata script resolves duplicate metadata records for the same strain name by selecting the record with the latest database id. Multiple database id columns allow the script to resolve duplicates when one or more columns has ambiguous values (e.g., “?”). Deduplication occurs before renaming of columns, so the default values include GISAID’s own “Accession ID” as well as Nextstrain-style database ids.
+-  default:
+
+.. code:: yaml
+
+     - "Accession ID"
+     - gisaid_epi_isl
+     - genbank_accession
+
+error_on_duplicate_strains
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  type: boolean
+-  description: Exit the sanitize metadata script with an error when any strains have multiple records in the metadata. The script writes list of all duplicate strains to a file named like ``<input>.duplicates.txt`` that users can review and use to address unexpected duplicates.
+-  default: ``false``
+
+parse_location_field
+~~~~~~~~~~~~~~~~~~~~
+
+-  type: string
+-  description: Field in the metadata that stores GISAID-formatted location details (e.g., ``North America / USA / Washington``) to be parsed into ``region``, ``country``, ``division``, and ``location`` fields.
+-  default: ``Location``
+
+rename_fields
 ~~~~~~~~~~~~~
 
+-  type: array
+-  description: List of key/value pairs mapping fields in the input metadata to rename to another value in the sanitized metadata.
+-  default:
+
+.. code:: yaml
+
+       - "Virus name=strain"
+       - "Type=type"
+       - "Accession ID=gisaid_epi_isl"
+       - "Collection date=date"
+       - "Additional location information=additional_location_information"
+       - "Sequence length=length"
+       - "Host=host"
+       - "Patient age=patient_age"
+       - "Gender=sex"
+       - "Clade=GISAID_clade"
+       - "Pango lineage=pango_lineage"
+       - "Pangolin version=pangolin_version"
+       - "Variant=variant"
+       - "AA Substitutions=aa_substitutions"
+       - "aaSubtitutions=aa_substitutions"
+       - "Submission date=date_submitted"
+       - "Is reference?=is_reference"
+       - "Is complete?=is_complete"
+       - "Is high coverage?=is_high_coverage"
+       - "Is low coverage?=is_low_coverage"
+       - "N-Content=n_content"
+       - "GC-Content=gc_content"
+
+
+
+combine_sequences_for_subsampling
+---------------------------------
+
+-  type: object
+-  description: Configuration of logic to combine sequences from multiple input files into a single file for subsampling.
+
+warn_about_duplicates
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
> @jameshadfield For instance, can you tell that "warn_about_duplicates" is a parameter of the parent config option "combine_sequences_for_subsampling"?

Yeah, this is something that can be improved. The individual `.. contents::` helps a bit for `<h3>`s which have a lot of `<h4>`s, such as `build`. I didn't add one here since there is only one `<h4>` but maybe a description + `.. contents::` here would help.

Reply:
Ahh, those mini-TOCs are really good. I'll go through and see if I can add them anywhere.

I tried playing around with encoding the structure rather than just the variable name, but I'm not sure what syntax to use (I presume you'd follow how type-hints are done in python?) and I'm not sure readers would find it clearer.

![image](https://user-images.githubusercontent.com/8350992/162335090-a84b0099-d59d-4b98-88c9-5addd35be366.png)

Reply:
Added styles in [743712d](https://github.com/nextstrain/ncov/pull/907/commits/743712d379b02907db382710d89c56db9b060776) which I think fix this :)